src/pages/dashboard/index.tsx

```diff
@@ -4,11 +4,6 @@ import PageNotFound from '../404';
 import NavBar from '@/components/navBar';
 import Searchbar from '@/components/Dashboard/Searchbar';
 
-const search = (query: string) => {
-    const searchValues = query.split(',');
-    console.log('Searching', searchValues);
-};
-
 const DashboardPage = () => {
```

Comment:
The dev property in the router's query is only set to true when the application is running in development mode. In production mode, the dev property will be undefined. This means that the if statement will always evaluate to false, and the PageNotFound component will always be rendered.

Reply:
i want something like below

const DashboardPage = () => {
    const router = useRouter();

    const { dev } = router.query;
    if (typeof dev === "undefined") {
        return <PageNotFound />;
    }
    return (
        <>
            <NavBar />
            <Searchbar label="Users" />
        </>
    );
};

Reply:
it's not based on the URL being dev.realdevsquad.com

it is the query param that we pass in the URL. something like this `/?dev=true`